appserver/registration/glassfish-registration/src/main/java/com/sun/enterprise/registration/glassfish/ModuleMap.java

```diff
@@ -159,7 +159,7 @@
                 put("org.glassfish.hk2.config", 100);
                 put("org.glassfish.hk2.config-types", 101);
                 put("org.glassfish.hk2.core", 102);
-                put("org.glassfish.hk2.external.asm-repackaged", 103);
+                //put("org.glassfish.hk2.external.asm-repackaged", 103);
```

Comment:
Shouldn't these be removed? (also the ones already there?) Or is there a particular reason to keep them?